worker/pkg/decompressor/archive.go

```diff
@@ -0,0 +1,16 @@
+package decompressor
+
+import (
+	"io"
+)
+
+type file interface {
```

Comment:
```suggestion
type File interface {
```

Public interface is OK, it's very simple interface.